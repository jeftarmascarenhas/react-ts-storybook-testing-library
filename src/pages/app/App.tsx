import React from "react";
// import logo from './logo.svg';

const App: React.FC = () => {
  return (
    <div>
      <h2>Run command `yarn storybook` for you see all components :(</h2>
    </div>
  );
};

export default App;
