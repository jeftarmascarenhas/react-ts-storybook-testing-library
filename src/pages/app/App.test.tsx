import React from "react";
import { render } from "@testing-library/react";

import App from "./App";

test("App Rendering", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});

test("renders learn react link", () => {
  const { getByText } = render(<App />);
  const linkElement = getByText(/yarn storybook/i);
  expect(linkElement).toBeInTheDocument();
});
