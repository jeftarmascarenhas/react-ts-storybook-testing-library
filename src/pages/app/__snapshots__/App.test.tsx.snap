// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Rendering 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Run command \`yarn storybook\` for you see all components :(
    </h2>
  </div>
</DocumentFragment>
`;
